src/app/portlets/dot-browser/dot-browser.module.ts

```diff
@@ -3,57 +3,39 @@ import { CommonModule, LocationStrategy, HashLocationStrategy } from '@angular/c
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 
 import { DotBrowserComponent } from './dot-browser-component';
-import {
-    // COMPONENTS
-    SiteSelectorComponent,
-    DotcmsBreadcrumbModule,
-    DotcmsSiteTreeTableModule,
-    DotcmsSiteDatatableModule,
-    DotcmsTreeableDetailModule,
-    // SERVICES
-    AppConfig,
-    FileService,
-    HttpClient,
-    LocalStoreService,
-    LoggerService,
-    NotificationService,
-    SettingsStorageService,
-    SiteBrowserService,
-    SiteBrowserState,
-    SiteSelectorService,
-    SiteTreetableService
-} from '../../../dotcms-js';
 
 import {AutoCompleteModule} from 'primeng/primeng';
+import {
+  DotBreadcrumbModule, DotFileModule, DotFolderModule, DotHttpModule, DotNotificationModule, DotSettingsStorageModule,
```

Comment:
Double check indentation here, seems there is a combination of 2 and 4 spaces in this file.